File: code/python/sum_arguments.py
#!/usr/bin/python
# writen by andrewt@cse.unsw.edu.au as a COMP2041 example
# sum integers supplied as command line arguments
# no check that arguments are integers

import sys

sum = 0
for arg in sys.argv[1:]:
    sum += int(arg)
print("Sum of the numbers is %s" % sum)

File: code/perl_arrays/sum_arguments.pl
#!/usr/bin/perl -w
# writen by andrewt@cse.unsw.edu.au as a COMP2041 example
# sum integers supplied as command line arguments
# no check that aguments are numeric

$sum = 0;
foreach $arg (@ARGV) {
	$sum += $arg;
}
print "Sum of the numbers is $sum\n";
